README.md

```diff
@@ -64,51 +64,20 @@ Android Studio - View > Tool Windows > Build Variants ... In the Build Variants
 
 Open `app/app.js` to get started with development on this project!
 
+## Quick Configuration
+
+The scaffold project contains configuration files located under `app/app-config/` that you can modify for quick customizations.
+
+## Enabling Mobify Preview
+
+To enable Mobify Preview for Adaptive.js and Mobify.js projects, go into the `baseConfig.js` file, set `previewEnabled = true`, and specify a bundle to preview `previewBundle` (i.e. http://localhost:8443/adaptive.js).
+
 ## Controller Architecture
 
 Astro uses a controller pattern to manage the business logic of UI components. Controllers can manage a single Astro plugin or coordinate the behavior of several astro plugins which make up a single UI component in the app.
 
 Astro defines a controller inside of a requirejs module. The module exposes a prototype object as well as a factory method (`init`) which can be used to instantiate controllers with the prototype. The `init` method returns a promise which resolves to the newly created object.
 
-## App Structure
```

Comment:
cool, I have some of this plus more on this branch: https://github.com/mobify/astro-scaffold/compare/readme-cleanup

However, I haven't gotten around to finishing yet so seems good. I figured I should discuss more before removing circle so I back-burnered it for now. Let's leave yours as is though because these changes are definitely not controversial.